scipy/sparse/linalg/eigen/arpack/tests/test_arpack.py

```diff
@@ -136,26 +136,26 @@ def assert_allclose_cc(actual, desired, **kw):
         assert_allclose(actual, conj(desired), **kw)
 
 
-def argsort_which(eval, typ, k, which,
+def argsort_which(eigenvalues, typ, k, which,
                   sigma=None, OPpart=None, mode=None):
     """Return sorted indices of eigenvalues using the "which" keyword
     from eigs and eigsh"""
     if sigma is None:
-        reval = np.round(eval, decimals=_ndigits[typ])
+        reval = np.round(eigenvalues, decimals=_ndigits[typ])
     else:
         if mode is None or mode == 'normal':
             if OPpart is None:
-                reval = 1. / (eval - sigma)
+                reval = 1. / (eigenvalues - sigma)
             elif OPpart == 'r':
```

Comment:
Is there any chance `OPpart` is none of these options to catch?

Reply:
I agree, but this is actually unrelated to the PR. I only renamed "eval" into "eigenvalues" to make the pylint happier.  Also, this file is only a test, where I am sure there are issues like this one in many-many spots...

Reply:
Please advise?

Reply:
I think no change is needed here (the test would just run into NameError)

Reply:
Thanks! Mark as resolved